onnxruntime/test/providers/tensorrt/tensorrt_basic_test.cc

```diff
@@ -17,6 +15,30 @@ using namespace ::onnxruntime::logging;
 namespace onnxruntime {
 
 namespace test {
+namespace {
+#ifndef _WIN32
```

Comment:
why is this code duplicated if it's already in scoped_env_vars.cc ? 
just include it in the compilation of this test.